jobs/e2e-test-cypress.groovy

```diff
@@ -145,8 +201,19 @@ onWorker("ka-test-ec2", '6h') {
       stage("Sync webapp") {
          _setupWebapp();
       }
-      stage("Run e2e tests") {
-         runLamdaTest();
+
```

Comment:
**note:** this logic is pretty similar to the Python/Selenium one.

See: https://github.com/Khan/jenkins-jobs/blob/777e1934c8204b466b01cdb51a832f5694181bc8/jobs/e2e-test.groovy#L604-L639